captcha.js

```diff
@@ -1,45 +1,42 @@
-const Canvas = require('canvas');
+const Canvas = require('canvas')
+const fs = require('fs')

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nidrux/simple-captcha-generator/pullRequest?prid=7734724)
```suggestion
const fs = require("fs")
```